project/profiles/capacity4more/modules/c4m/widgets/c4m_document_widget/c4m_document_widget.module

```diff
@@ -475,7 +475,7 @@ function _c4m_document_widget_document_file($file_id, $is_quick_post = FALSE) {
   // PURL options.
   $options_context = array(
     'purl' => array(
-      'disabled' => FALSE,
+      'enabled' => TRUE,
```

Comment:
This change does not solve the problem in my case.
Tested by attaching a document while creating a new event in group nobelprize as alfrednobel (GO). 
I will collect the gid from the post data in the C4MOgSelectionHandler class.